contracts/HubPool.sol

```diff
@@ -286,7 +371,18 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
      *          LIQUIDITY PROVIDER FUNCTIONS         *
      *************************************************/
 
-    function addLiquidity(address l1Token, uint256 l1TokenAmount) public payable {
+    /**
+     * @notice Deposit liquidity into this contract to earn LP fees in exchange for backstopping relays on SpokePools.
```

Comment:
is backstopping the right word? I get what you're saying; it totally makes sence but is it the most correct word?

Reply:
Yeah, I think funding is probably a more correct word.

Reply:
changed to funding